shared/js/media/video_player.js

```diff
@@ -237,6 +243,9 @@ function VideoPlayer(container) {
   // Set the video duration when we get metadata
   player.onloadedmetadata = function() {
     durationText.textContent = formatTime(player.duration);
+    slider.setAttribute('aria-valuemax', player.duration);
+    navigator.mozL10n.setAttributes(slider, 'playbackSeekBar',
+                                    {'duration': formatTime(player.duration)});
```

Comment:
Could you add a comment on the setAttributes() line to say that this is actually setting the aria-label attribute? Otherwise it is completely unclear what is going on here without referring to the locale file.  Is that actually what this line is doing?